Source/Core/VideoBackends/Vulkan/CommandBufferManager.cpp

```diff
@@ -18,6 +18,10 @@ namespace Vulkan
 CommandBufferManager::CommandBufferManager(bool use_threaded_submission)
     : m_use_threaded_submission(use_threaded_submission)
 {
+  for (u32 i = 0; i < m_frame_resources.size(); i++)
```

Comment:
Seems like this could be:

```suggestion
  for (auto& frame_resource : m_frame_resources)
```